src/test/kotlin/org/rust/lang/core/completion/RsAttributeCompletionTest.kt

```diff
@@ -221,4 +221,24 @@ class RsAttributeCompletionTest : RsCompletionTestBase() {
         #[track/*caret*/]
         struct Foo;
     """)
+
+    fun `test non_exhaustive on enum`() = doSingleCompletion("""
+        #[non_/*caret*/]
+        enum Foo {}
+    """, """
+        #[non_exhaustive/*caret*/]
+        enum Foo {}
+    """)
+
+    fun `test non_exhaustive on struct`() = doSingleCompletion("""
```

Comment:
Let's add two more tests: for enum variant and for some item (e.g. function) where this attribute shouldn't be completed